cmd/veneur-emit/main.go

```diff
@@ -224,9 +242,25 @@ func destination(hostport *string, useSSF bool) (string, net.Addr, error) {
 	return addr, netAddr, nil
 }
 
+func inferTraceIDInt(id *int64, envKey string) (err error) {
+	if id != nil && *id != 0 {
+		return nil // nothing to do
+	}
+	if strID, ok := os.LookupEnv(envKey); ok {
+		*id, err = strconv.ParseInt(strID, 10, 64)
+		if err != nil {
+			return
+		}
+		logrus.WithField("env_var", envKey).
```

Comment:
Nit: can we use logrus.WithFields here? That's more consistent with what we do elsewhere.